libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -252,10 +252,18 @@ singleton RangeFinder rename rangefinder
 singleton RangeFinder method num_sensors uint8_t
 singleton RangeFinder method has_orientation boolean Rotation'enum ROTATION_NONE ROTATION_MAX-1
 singleton RangeFinder method distance_cm_orient uint16_t Rotation'enum ROTATION_NONE ROTATION_MAX-1
+singleton RangeFinder method distance_cm_orient deprecate Use distance_orient (in metres)
```

Comment:
Nice! There is also some markup we should add to the docs. If you re-generate it might do that for you..... If not Just add
```
---@deprecated -- Use distance_orient (in metres)
```

https://luals.github.io/wiki/annotations/#deprecated

Reply:
Looks like that will update the static checkers, so you will also have disable that check on any files that need it with:
```
---@diagnostic disable deprecated
```
Or move them onto the new method of course....

Reply:
> Nice! There is also some markup we should add to the docs. If you re-generate it might do that for you..... If not Just add
> 
> ```
> ---@deprecated -- Use distance_orient (in metres)
> ```
> 
> https://luals.github.io/wiki/annotations/#deprecated

Done

Reply:
> Looks like that will update the static checkers, so you will also have disable that check on any files that need it with:
> 
> ```
> ---@diagnostic disable deprecated
> ```
> 
> Or move them onto the new method of course....

I have moved them to the new method - which I really should have done already!